react/Reviews.js

```diff
@@ -13,7 +13,7 @@ import ReviewsContainer from './components/ReviewsContainer'
 import queryRatingSummary from './graphql/queries/queryRatingSummary.gql'
 import getConfig from './graphql/getConfig.gql'
 import { withApollo, graphql } from 'react-apollo'
-import PropTypes from 'prop-types'
+import { trackPageViewData } from './components/utils/trackers'
```

Comment:
Move this file to a folder called `modules`

```suggestion
import { trackPageViewData } from './modules/trackers'
```